src/pages/portfolio-page/index.tsx

```diff
@@ -0,0 +1,152 @@
+      height: 668,
+      width: 280,
+      marginRight: theme.spacing(4),
+    },
+  },
+  main: {
+    [theme.breakpoints.up('lg')]: {
+      width: '100%',
+    },
+  },
+  mainContent: {
+    backgroundColor: 'transparent',
+    borderRadius: 16,
+    margin: theme.spacing(0, 3, 7, 3),
+
+    [theme.breakpoints.up('sm')]: {
+      backgroundColor: theme.palette.background.paper,
+    },
+
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      flexDirection: 'column',
+      margin: theme.spacing(0, 0, 7, 0),
+      boxShadow: '0 40px 50px 0 rgba(103, 118, 128, 0.1)',
+    },
+  },
+  navbar: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    boxShadow: '0 40px 50px 0 rgba(103, 118, 128, 0.1)',
+    borderRadius: 8,
+    height: 120,
+    width: '100%',
+    marginBottom: theme.spacing(2),
+
+    [theme.breakpoints.up('sm')]: {
+      marginBottom: theme.spacing(4),
+    },
+
+    [theme.breakpoints.up('lg')]: {
+      marginBottom: theme.spacing(7),
+      height: 56,
+    },
+  },
+  projectsContainer: {
+    backgroundColor: 'transparent',
+    borderRadius: 16,
+    padding: theme.spacing(3),
+
+    [theme.breakpoints.up('lg')]: {
+      width: '100%',
+      padding: theme.spacing(4),
+    },
+  },
+  projects: {
+    display: 'grid',
+    gridTemplateColumns: '1fr',
+    gridRowGap: theme.spacing(4),
+
+    [theme.breakpoints.up('sm')]: {
+      gridRowGap: theme.spacing(3),
+    },
+
+    [theme.breakpoints.up('lg')]: {
+      gridTemplateColumns: '400px 400px',
+      alignItems: 'center',
+      justifyContent: 'center',
+      gridGap: theme.spacing(5),
+    },
+  },
+  project: {
+    [theme.breakpoints.up('lg')]: {
+      width: 400,
+      height: 224,
+    },
+  },
+  title: {
+    marginBottom: theme.spacing(4),
+
+    [theme.breakpoints.up('sm')]: {
+      marginBottom: theme.spacing(3),
+    },
+
+    [theme.breakpoints.up('lg')]: {
+      margin: theme.spacing(0, 2, 4, 2),
+    },
+  },
+}));
+
+const IndexPage = () => {
+  const classes = useStyles();
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('xs'));
+  const isTablet = useMediaQuery(theme.breakpoints.down('md'));
+  const isDesktop = !isMobile && !isTablet;
+
+  return (
+    <Container className={classes.container} maxWidth="lg">
+      <Helmet>
+        <title>Portfolio page</title>
+      </Helmet>
+      {isDesktop && (
+        <Box className={classes.aside}>
+          <DetailsCard type="desktop" {...userData} />
+        </Box>
+      )}
+      <Box className={classes.main}>
+        <Card className={classes.navbar}>navbar</Card>
+        <Box className={classes.mainContent}>
+          <Box className={classes.projectsContainer}>
+            <SectionTitle className={classes.title}>My works</SectionTitle>
+            <Box className={classes.projects}>
+              {[1, 2, 3, 4, 5, 6].map((_) => (
+                <Box key={_} className={classes.project}>
```

Comment:
Do you have to use Box here? Is it possible to pass this className directly to the PortfolioCard component?

Reply:
Currently, it's not but I can adjust the component